test/micro/org/openjdk/bench/java/lang/foreign/pointers/NativeType.java

```diff
@@ -56,14 +58,14 @@ public non-sealed static abstract class OfStruct<X> extends NativeType<X> {
     public static final OfInt<Integer> C_INT = new OfInt<>() {
         @Override
         public ValueLayout.OfInt layout() {
-            return ValueLayout.JAVA_INT;
+            return (ValueLayout.OfInt) LINKER.canonicalLayouts().get("int");
```

Comment:
For this benchmark, we rely on constant folding through these layouts. So, we shouldn't go through `canonicalLayouts` here. We can store the layout in a `static final` field, and then return it here instead.